subprojects/core/src/main/java/org/gradle/api/internal/tasks/TaskStateInternal.java

```diff
@@ -122,4 +123,12 @@ public boolean getNoSource() {
     public boolean isFromCache() {
         return outcome == TaskExecutionOutcome.FROM_CACHE;
     }
+
+    public boolean isActionsWereExecuted() {
```

Comment:
This reads pretty gnarly... I'm assuming we're keeping the 'is' to make the groovy syntax easier?

Could we have another method just for readability without the is?